5.1.thats_the_way.py

```diff
@@ -0,0 +1,12 @@
+import fnmatch
+import os
+
+
+def deepfiles(path):
+    """
+     Return a list of files in `path` starting with "deep".
```

Comment:
Use type hints and return types -
https://www.geeksforgeeks.org/duck-typing-in-python/